spot-oa/oa/components/iana/dns-qclass.csv

```diff
@@ -6,5 +6,6 @@ Decimal,Hexadecimal,Name,Reference
 Technology Artificial Intelligence Laboratory, June 1981.]"
 4,0x00000004,Hesiod (HS),"[Dyer, S., and F. Hsu, ""Hesiod"", Project Athena Technical
 Plan - Name Service, April 1987.]"
+5,unknown,UNKNOWN,
```

Comment:
It's part of the changes. So, sometimes DNS records will come with _null_ query class, in order to have spot-ml to complete I need to assign a value to that null column, in this case unknown. Once it goes to dns_results.csv, OA needs to handle that value and present a proper label in UI.